.gitignore

```diff
@@ -1,6 +1,5 @@
+**/__pycache__
```

Comment:
Doesn't seem like this is working, as __pycache__ is still showing in your root directory. I can't emphasize enough how unsightly these are, so do put some effort into hiding them all.